app/controllers/application_controller.rb

```diff
@@ -1,2 +1,21 @@
 class ApplicationController < ActionController::Base
+  before_action :set_current_user
+
+  private
```

Comment:
![image](https://github.com/sumon766/techbright/assets/93039359/e571d0cb-081d-4695-9d29-26e91b906202)

- [ ] It would be nice if you double-check the requirements because there should be a form to create a post but I didn't find it anywhere in your posts_controller.rb file 😨 Kindly add the method and form for adding a post to make your project follow the requirements 👍